docker-compose.yml

```diff
@@ -29,4 +30,5 @@ services:
       - start
       - --host-port=0.0.0.0:5010
     ports:
-      - "${EXPOSE_GCS_EMULATOR_PORT:-5010}:5010"
+      - "${EXPOSE_PUBSUB_EMULATOR_PORT:-5010}:5010"
```

Comment:
noticed while editing that this was using the wrong env var name, this is the pubsub emulator